auth/src/test/resources/logback-test.xml

```diff
@@ -7,7 +7,7 @@
     </encoder>
     <target>System.err</target>
   </appender>
-  <root level="WARN">
+  <root level="DEBUG">
```

Comment:
Leave at `WARN` so that the tests aren't super verbose when we don't need them to be?